### PR TITLE
#Migrate to Next js

### PR DESCRIPTION
**Pull request by codingknite**
https://user-images.githubusercontent.com/70036189/115126052-007d6380-9fd5-11eb-9694-dc8c525e4ec6.mp4

### CODE REVIEW

File: pages/index.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  return (
+    <>
+      <Head>
+        <title>MovieHouse</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        <section>
+          <FeaturedMovie movie={featured} />
```
Comment (ryanmagoon) [inline pages/index.jsx:65]: this is kinda confusing cause it looks like you're taking in an array but counting on the array to only have one value? the plural vs singular vars and all that make it kinda hard to follow what you're trying to do so it would probably be work tweaking the var names to make it clearer what's going on

File: components/DiscoverGenres/index.jsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
-import { Link } from "react-router-dom";
 import { useQuery } from "react-query";
-import Slider from "react-slick";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
 
 import fetchData from "../../utils/fetchData";
 import { genres } from "../../data/categories";
-import MovieCards from "../MovieCards/MovieCards";
+import MovieCards from "../MovieCards/index";
```
Comment (ryanmagoon) [inline components/DiscoverGenres/index.jsx:10]: You should be able to import without adding the `/index` to the end of the path. it should automatically find the index.js/jsx/tsx file

File: components/Layout.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export default function Layout({ children }) {
+  return (
+    <>
```
Comment (ryanmagoon) [inline components/Layout.jsx:5]: extra fragment 🤷🏻‍♂️

**Review by ryanmagoon — APPROVED** [review]
Overall looking pretty good, I would say in general look at where you might be adding divs for no reason or cases where it might be better to have a semantic HTML5 tag like `section` or `header` or something like that

File: components/MovieCards/index.jsx [hunk 4]
```diff
@@ -1,35 +1,36 @@
...
+                <Lazyload>
+                  <div className="poster">
+                    <img
+                      src={posterUrl + movie.poster_path}
+                      className="movie-banner"
+                      alt={movie.title}
+                      onLoad={() => setImageLoaded(true)}
+                    />
+                  </div>
+                  <div>
```
[the remark this answers is not in the document]
Comment (codingknite) [reply components/MovieCards/index.jsx:29]: Yeah actually..,one of those is the container and the other is a child in the container

File: components/Biography/index.jsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+                </p>
+            </div>
+        </section>
+    );
+}
+
+export function BioData({ biography, imdbLink, homepage }) {
+    return (
+        <section>
+            <p>{biography ? biography : <h3>Biography Not Found</h3>}</p>
```
Comment (ryanmagoon) [inline components/Biography/index.jsx:26]: if you want you can do `biography || <h3>...</h3>`

File: components/DiscoverGenres/index.jsx [hunk 6]
```diff
@@ -38,7 +39,7 @@ export default function DiscoverGenres() {
   return (
     <section>
       <h2>Movies</h2>
-      <GenreCards>
+      <div>
```
Comment (ryanmagoon) [inline components/DiscoverGenres/index.jsx:42]: do you need this div?

File: components/NavBar/index.jsx [hunk 7]
```diff
@@ -1,97 +1,95 @@
...
             src="https://fontmeme.com/permalink/210413/c070bb97de43f148099652bce2e916fc.png"
             alt="logo"
+            height={100}
+            width={350}
           />
         </div>
 
-        <NavItems>
-          <NavItem activeClassName="active" to="/" end>
+        <ul>
```
Comment (ryanmagoon) [inline components/NavBar/index.jsx:26]: `ul` with no `li`?

**Review by ryanmagoon — APPROVED** [review]
Looking great! Let me know if there's a particular part you want me to look at more closely.
